superset/assets/visualizations/deckgl/layers/geojson.jsx

```diff
@@ -70,14 +67,16 @@ export default function geoJsonLayer(formData, payload, slice) {
     propOverrides.strokeColor = strokeColor;
   }
 
+  features = [];
+  recurseGeoJson(payload.data, propOverrides);
+
   let jsFnMutator;
-  if (fd.js_datapoint_mutator) {
+  if (fd.js_data_mutator) {
```

Comment:
nit: later on we should have some type of baseComponentClass for the components so we aren't copying this condition into each layer. 

What are your thoughts? @mistercrunch